k8s/demo/common/money-claims/cmc-demo.yaml

```diff
@@ -35,8 +35,8 @@ spec:
       draftStoreUrl: "http://cmc-draft-store-java"
       s2sAuthUrl: "http://cmc-s2s-java"
       pdfServiceUrl: "http://pdf-service-java"
-      paymentUrl: "http://payment.demo.platform.hmcts.net"
-      feesUrl: "http://fees-register-api.demo.platform.hmcts.net"
+      paymentUrl: "http://payment-api-demo.service.core-compute-demo.internal"
```

Comment:
shouldn't you also remove:
https://github.com/hmcts/cnp-flux-config/blob/repointingtoasedemo/k8s/demo/common/money-claims/cmc-pay.yaml

Reply:
Let it be there temporarily, till they test and see it's working.

Reply:
k